eng/build.sh

```diff
@@ -524,6 +524,16 @@ while [[ $# > 0 ]]; do
       shift 2
       ;;
 
+      -skipcrossarchnative)
```

Comment:
This argument no longer does anything. It might be better to remove it from the help.

Reply:
@hoyosjs thanks for reviewing. You're right, but I also found a left-over here:
https://github.com/dotnet/runtime/blob/main/src/coreclr/build-runtime.sh#L23
I can add a patch to remove it in this PR in place of the skipcrossarchnative patch Is it ok for your?